controllers/apply/awards-for-all/form.js

```diff
@@ -799,7 +826,12 @@ module.exports = function({
                             We can't transfer money into certain types of bank
                             accounts like Tide, Cashplus and Paypal.
                         </p>`,
-                        cy: ''
+                        cy: `<p>
+                            Rydym angen eich manylion banc i dalu’r arian i’ch cyfrif – os yw eich cais yn llwyddiannus. 
+                        </p>
+                        <p><strong>We can't pay into all bank accounts</strong>:
```

Comment:
This line was left as English in the Welsh translation ie. "We can't pay into all bank accounts" was not translated.

Reply:
TODO – we should fix this before merging as we'll definitely forget :D

Reply:
Have just requested this. Worked out why this was missed, the text was emboldened and we asked for bold text to be ignored :)